L1Trigger/Phase2L1ParticleFlow/src/BitwisePFAlgo.cc

```diff
@@ -0,0 +1,204 @@
+#include "L1Trigger/Phase2L1ParticleFlow/interface/BitwisePFAlgo.h"
+#include "FWCore/Utilities/interface/Exception.h"
+
+//#define REG_HGCal
+#include "ref/pfalgo2hgc_ref.h"
```

Comment:
having headers inside subfolders in the src directory is unusual. why aren't these just in interface?

Reply:
isn't it better to have them under `src1 than under `interface` if they are only used inside the package, and not part of its public interface ?

Reply:
indeed these are header files for things that shouldn't be used from outside the package (and they look as they do becuase they're used also in the unit tests of the firmware run by Vivado outside CMSSW)